plugins/src/lib.rs

```diff
@@ -210,6 +212,17 @@ where
         self
     }
 
+    /// Sets the "featurebits" in the "getmanifest" response
+    pub fn featurebits(mut self, space: FeatureBitsPlace, hex: String) -> Self {
+        match space {
+            FeatureBitsPlace::Node => self.featurebits.node = Some(hex),
+            FeatureBitsPlace::Channel => self.featurebits.channel = Some(hex),
+            FeatureBitsPlace::Init => self.featurebits.init = Some(hex),
+            FeatureBitsPlace::Invoice => self.featurebits.invoice = Some(hex),
+        }
+        self
+    }
```

Comment:
Just a nit: you use `space` for a parameter of type `FeatureBitsPlace`, calling it `place` seems more consistent but I am not going to fight for this :D

Reply:
Fixed this in the latest replace.